experiment/manage_release_versions.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
```

Comment:
was there a requirement for this to be in python?
i think the general consensus is for new tools in this repo to be in GO and move away from the old py code.
cc @krzyzacy @fejta

Reply:
This was discussed and I was suggested to use python since most of the existing tools are in python. 
@cjwagner @amwat